Formula/jenkins-lts.rb

```diff
@@ -1,8 +1,8 @@
 class JenkinsLts < Formula
   desc "Extendable open-source CI server"
   homepage "https://jenkins.io/index.html#stable"
-  url "https://get.jenkins.io/war-stable/2.289.1/jenkins.war"
-  sha256 "70f9cc6ff1ac59aeeb831b980709a9ddb0ee70d216ee50625a8508b9840f75f2"
+  url "http://mirrors.jenkins.io/war-stable/2.289.2/jenkins.war"
```

Comment:
What is reason for switching to HTTP url?

From download page (https://www.jenkins.io/download/), the previous URL format is linked
https://get.jenkins.io/war-stable/2.289.2/jenkins.war

Reply:
It was using mirrors before (actually for `jenkins` formula it was still using `mirrors` url). Just corrected the change.